canonical-web-and-design/ubuntu.com
# Add advantage beta form

## Done
- Add advantage beta interactive form
- Added a small notification to the top of the page to call users to join the beta

## QA
- Go to /advantage/subscribe
- See that the notification content matches the [content from the brief](https://docs.google.com/document/d/1NoXtzQniQ-puVggi1JDcjUEeVVngKtlmFB86u_FYWBo/edit)
- Click the link
- Check the interactive form matches the [content for the form](https://docs.google.com/document/d/1qrJVP2C1l36WLHRXtBMtF9BwzIkAdLXoKmoNmZZd3Lw/edit#heading=h.dxn10eelvkoc)
- Submit the form and check there are no errors and the XHR request to save2 endpoint contains the form info in the request payload
- I have checked the submission is getting to Marketo

## Issue / Card
Fixes https://github.com/canonical-web-and-design/web-squad/issues/3905

## Screenshots
![image](https://user-images.githubusercontent.com/1413534/110487557-9a541580-80e5-11eb-9ff0-c6505e27e796.png)
